samples/updatehub/README.rst

```diff
@@ -0,0 +1,246 @@
+UpdateHub sample
+################
+
+Overview
+********
+
+These are samples on how to implement simple, polling or manual, local update
+using UpdateHub. UpdateHub is an enterprise-grade solution which makes simple
+to remotely update all your Linux-based devices in the field. It handles all
+aspects related to sending Firmware Over-the-Air (FOTA) updates with maximum
+security and efficiency, while you focus in adding value to your product.
+
+In this document you will find two modes of UpdateHub operation: polling and
+manual
+
+Polling mode runs automatically on a predefined period, probing the server
+for updates and installing them without requiring user intervention. You
+can access the sample source code for this mode `updatehub_polling`_.
+
+.. _updatehub_polling: updatehub_polling
```

Comment:
I'm guessing you'll provide a real external link to where this update polling content is?  As written this won't work.